### ISSUE TITLE
BTCV train

### ISSUE THREAD

**Issue by 6018203135** [ge-xing/Diff-UNet#28]
Hello, how is the BTCV training set divided, and can you directly use the original dataset without data preprocessing?
I use the list of BTCV training datasets below and get an error.
![btcv](https://github.com/ge-xing/Diff-UNet/assets/50786422/6cab6ce7-c364-48ee-acfb-e431de357513)
![btcv错误](https://github.com/ge-xing/Diff-UNet/assets/50786422/19865f0f-296a-480d-ad96-bffb2d28924a)

**Comment by 920232796** [comment]
all_images = sorted(glob.glob(f"{data_dir}imagesTr/*.nii.gz"))
    all_labels =  sorted(glob.glob(f"{data_dir}labelsTr/*.nii.gz"))
    
    Please modify the directory "img" to "imagesTr", "label" to "labelsTr"
